pom.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.github.Ericsson</groupId>
+        <artifactId>eiffel-remrem-parent</artifactId>
+        <version>0.0.2</version>
+    </parent>
+    <properties>
+    <eiffel-remrem-publish.version>0.6.2</eiffel-remrem-publish.version>
+    <eiffel-remrem-shared.version>0.3.6</eiffel-remrem-shared.version>
+    <eiffel-remrem-semantics.version>0.4.4</eiffel-remrem-semantics.version>
+    </properties>
+    <artifactId>eiffel-remrem-publish</artifactId>
+    <version>${eiffel-remrem-publish.version}</version>
+    <packaging>pom</packaging>
+	<modules>
+		<module>publish-common</module>
+		<module>publish-service</module>
+		<module>publish-cli</module>
+	</modules>
+	<repositories>
+		<repository>
+			<id>jitpack.io</id>
+			<name>Jitpack Repository</name>
+			<url>https://jitpack.io/</url>
+		</repository>
```

Comment:
add the below repository also 
maven{
        url 'https://repo.spring.io/libs-milestone'
 }

Reply:
check xml formatting

Reply:
changes done